LEDs/StrutLayoutFactory.cs

```diff
@@ -48,6 +49,43 @@ public class StrutLayoutFactory {
     };
     private static readonly EdgeDictionary edgeDictionary;
 
+    // JKMD: these are all the points of the dome, mapped into 2d space
```

Comment:
btw, I found the name of this projection from the [source code](https://github.com/robbykraft/Domekit/blob/master/Domekit/Views/EquidistantAzimuthView.m) of the app I used to make the initial image. It's called an "Equidistant Azimuth" projection